backends/platform/ios7/ios7_osys_video.mm

```diff
@@ -502,9 +502,12 @@ static inline void execute_on_main_thread(void (^block)(void)) {
 	}
 }
 
-void OSystem_iOS7::setMouseCursor(const void *buf, uint w, uint h, int hotspotX, int hotspotY, uint32 keycolor, bool dontScale, const Graphics::PixelFormat *format) {
+void OSystem_iOS7::setMouseCursor(const void *buf, uint w, uint h, int hotspotX, int hotspotY, uint32 keycolor, bool dontScale, const Graphics::PixelFormat *format, const byte *mask) {
 	//printf("setMouseCursor(%p, %u, %u, %i, %i, %u, %d, %p)\n", (const void *)buf, w, h, hotspotX, hotspotY, keycolor, dontScale, (const void *)format);
 
+	//if (mask)
```

Comment:
Why is this commented out?

Reply:
There are not any other live printfs in that file so I'm unsure if it's usable.